### ISSUE TITLE
Particle Counts and Ratio information go to different places

### ISSUE THREAD

**Issue by terracoda** [phetsims/ph-scale#354]
When the "H 3 O Plus To O H Minus Ratio" checkbox is checked, the description of the ratio is found under the Beaker description.
<img width="393" height="191" alt="Image" src="https://github.com/user-attachments/assets/cd1fe8cf-5476-4d93-aa9f-37c626b1d623" />

When the "Particle Counts" checkbox is checked the list of counts is displayed under the checkbox, even though the counts are displayed over the solution in the beaker about the solution in the beaker.
<img width="830" height="430" alt="Image" src="https://github.com/user-attachments/assets/a705911d-4b53-4980-a7b0-eb9e83715dba" />

This is a good example of disorganized state descriptions that makes it difficult to have the organization of state descriptions meet Level A SC 1.3.1 Info and Relationships.

Ratio and particle counts are information about the solution in the beaker, and should be organized under the Beaker heading. The checkboxes are just there to hide or show the information to simplify the amount of information available, or to assist with predictive reasoning.

**Comment by terracoda** [comment]
The intent of the issue is forward thinking. In the future, we need to put info together that goes together.
